upgrading/automated_upgrades.adoc

```diff
@@ -326,102 +181,33 @@ the host you run the upgrade playbooks on:
-[nodes]
-master[1:3].example.com openshift_node_group_name='node-config-master'
-infra-node1.example.com openshift_node_group_name='node-config-infra'
-infra-node2.example.com openshift_node_group_name='node-config-infra'
-node1.example.com openshift_node_group_name='node-config-compute'
-node2.example.com openshift_node_group_name='node-config-compute'
-----
-+
-In addition, remove the `openshift_node_labels` setting from any existing host
-entries `[nodes]` group if they are set. Node labels should now be defined in
-the ConfigMap associated with the host's `openshift_node_group_name` instead.
-
-.. The upgrade process will be blocked until you have the newly required ConfigMaps
-for bootstrapping nodes in the *openshift-node* project. Run the following
-playbook to have the defaults created (or if you defined the
-`openshift_node_groups` structure in the previous step, your custom sets will be
-created):
-+
-[subs=attributes+]
-----
-# ansible-playbook -i </path/to/inventory/file> \
-    {pb-prefix}playbooks/openshift-master/openshift_node_group.yml
-----
-
-.. Carefully verify that each ConfigMap has created and defined as expected. Get
-your list of all ConfigMaps in the *openshift-node* project:
-+
-----
-$ oc get configmaps -n openshift-node
-----
-+
-Then use `oc describe` to inspect them individually:
-+
-----
-$ oc describe configmaps -n openshift-node <configmap_name>
-----
 
-. If you have applied manual configuration changes to your master or node
-configuration files since your last Ansible playbook run (whether that was
-initial installation or your most recent cluster upgrade), and you have not yet
-made the equivalent changes to your inventory file, review
-xref:../install/configuring_inventory_file.adoc#configuring-ansible[Configuring
-Your Inventory File]. For any variables that are relevant to the manual
+. Review and update your
+xref:../install/configuring_inventory_file.adoc#configuring-ansible[inventory file].
+.. Ensure that any manual configuration changes you made to your master or node
+configuration files since your last Ansible playbook run, whether that was
+initial installation or your most recent cluster upgrade, are in the inventory
+file. For any variables that are relevant to the manual
 changes you made, apply the equivalent appropriate changes to your inventory
-files before running the upgrade. Otherwise, your manual changes may be
+files before running the upgrade. Otherwise, your manual changes might be
 overwritten by default values during the upgrade, which could cause pods to not
 run properly or other cluster stability issues.
-+
-In particular, if you made any changes to `admissionConfig` settings in your
+.. To check if your certificates will expire, set the
+`openshift_certificate_expiry_warning_days` parameter value to the number of
+days that your certificate must valid. For example, to ensure that your
+certificates are good for a year, set
+`openshift_certificate_expiry_warning_days` to `365`.
```

Comment:
We could also add the usage of `openshift_certificate_expiry_warning_days` parameter here. 
Set openshift_certificate_expiry_warning_days to False could bypass the check.